plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestParquetReaderIgnoreStatistics.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.hive.parquet;
+
+import com.google.common.collect.ImmutableSet;
+import io.trino.plugin.hive.HiveConfig;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.hive.HiveTestUtils.getHiveSession;
+import static org.apache.parquet.column.ParquetProperties.WriterVersion.PARQUET_1_0;
+import static org.apache.parquet.format.CompressionCodec.GZIP;
+
+// Failing on multiple threads because of org.apache.hadoop.hive.ql.io.parquet.write.ParquetRecordWriterWrapper
+// uses a single record writer across all threads.
+// For example org.apache.parquet.column.values.factory.DefaultValuesWriterFactory#DEFAULT_V1_WRITER_FACTORY is shared mutable state.
+@Test(singleThreaded = true)
+public class TestParquetReaderIgnoreStatistics
```

Comment:
This test is quite heavy and doesn't test with filters, so I don't think this is appropriate for testing ignore statistics property. When should either add a test in TestParquetPageSourceFactory or add a test in BaseHiveConnectorTest which runs a query with filter using `parquet_ignore_statistics` session property.

Reply:
@raunaqmorarka updated the test. PTAL